style.css

```diff
@@ -0,0 +1,108 @@
+/* General CSS Rules */
+:root {
+  --blue: #6070ff;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+button {
+  cursor: pointer;
+  border: none;
+  background: none;
+}
+
+ul {
+  list-style: none;
+}
+
+/* Navbar Design */
+nav {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  background-color: white;
+  height: 8vh;
+  top: 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.links {
+  display: none;
+}
+
+.logo {
+  margin-left: 24px;
+}
+
+.logo a {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--blue);
+}
+
+.hamburger {
+  margin-right: 2vw;
+}
+
+/* Headline Design */
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 680px;
+  background-color: white;
+  margin-top: 8vh;
+  border-bottom-left-radius: 8em;
+  padding: 0 24px 10px 24px;
+  background-image: url(img/header-background.png);
+  background-position: 50% 50%;
+  background-repeat: no-repeat;
```

Comment:
- [ ] I commend your efforts in incorporating a background image into your project. 
- [ ] However, it seems that the chosen image does not align with the suggested [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=uPJ9iFJPQJqhsLSg-0) design. To create an appealing and visually cohesive project that captures the attention of future recruiters,
- [ ]  I recommend following the Figma design guidelines closely. This will ensure that your project looks good and leaves a positive impression on viewers.🤔🤔🤔
- [ ] your design: 
![yourD](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/118761303/dc468de7-151e-4c0b-b6c6-4a3ae6154317)
Figma design:
![figmaD](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/118761303/61c16efd-ee77-43da-84e4-2034d3607157)